models/AppsModel.js

```diff
@@ -0,0 +1,22 @@
+// 1.importing mongoose
+const mongoose = require('mongoose')
+const moment = require("moment");
+//date formating to display it with the time as well
+const date = new Date();
+const formattedDate = moment(date).format("YYYY-MM-DD HH:mm:ss");

```

Comment:
please move "YYYY-MM-DD HH:mm:ss" to the Constants file and please don't hard coding any more